packages/react-params/src/index.tsx

```diff
@@ -91,12 +93,20 @@ class Params extends React.PureComponent<Props, State> {
 
   public override render (): React.ReactNode {
     const { className, isDisabled, onEnter, onEscape, overrides, params, style } = this.props;
-    const { values = this.props.values, assetIdContext } = this.state;
+    const { values = this.props.values, assetIdContext, collectionId } = this.state;
 
     if (!values || !values.length) {
       return null;
     }
 
+    const collectionIdParam = params.find(({ name }) => name === 'CollectionId');
+
+    if (!!collectionId && !!collectionIdParam) {
+      const index = params.indexOf(collectionIdParam);
+
+      values[index].value = new BN(collectionId);
+    }
+
```

Comment:
Feel like we are only fixing `collectionId`. Are other params input work as expected?